randovania/games/fusion/logic_database/Sector 3 PYR.txt

```diff
@@ -548,13 +548,17 @@ Extra - room_id: [14]
   * L0 Hatch to Sova Suite/Door to Pyrochamber Access
   * Extra - door_idx: (28,)
   > Door to Pyrochamber
-      All of the following:
-          # TODO: Possibly add singlewall jump trick difficulty with jump extend as an option if singlewall jump is kept.
-          Any of the following:
-              Space Jump
-              All of the following:
-                  Can Freeze Enemies
-                  High Jump or Wall Jump (Beginner)
+      Any of the following:
+          Space Jump
+          All of the following:
+              Can Freeze Enemies
+              High Jump or Wall Jump (Beginner)
+          All of the following:
+              # Jump Extend SWJ Method
+              Jump Extends (Advanced) and Can Single Walljump
+              Any of the following:
+                  Wall Jump (Advanced)
```

Comment:
This block needs you being able to kill the fune's too. So Screw or maybe supers?
I'm unsure whether with HJ you can avoid killing the upper fune.

Reply:
Won't this SWJ block need SWJ and Screw attack. The SA req would obsolete the `Beam-Resistant Enemy` template.'